figures/serializers.py

```diff
@@ -105,9 +105,10 @@ class UserIndexSerializer(serializers.Serializer):
     """
     id = serializers.IntegerField(read_only=True)
     username = serializers.CharField(read_only=True)
-    fullname = serializers.CharField(
-        source='profile.name', default=None, read_only=True)
-
+    fullname = serializers.CharField(source='profile.name', default=None, read_only=True)
+    email = serializers.CharField(read_only=True)
+    date_joined = serializers.CharField(read_only=True)
```

Comment:
why are date fields set as CharField?

Reply:
It was a mistake, replaced by the Date field.